### PR TITLE
Update README.md and src/AppInstallerCLIE2ETests/AppInstallerCLIE2ETests.csproj

### PR DESCRIPTION
**Pull request by skanda890**
- [X] I have signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs).
- [ ] This pull request is related to an issue.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-cli/pull/4618)

### CODE REVIEW

**Comment by skanda890** [comment]
Meaning of style cops?

**Comment by florelis** [comment]
> Meaning of style cops?

[StyleCop](https://github.com/DotNetAnalyzers/StyleCopAnalyzers) is a thingy to enforce consistent source code style

File: src/AppInstallerCLIE2ETests/AppInstallerCLIE2ETests.csproj [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
   <PropertyGroup>
     <!-- Disable warning for CS1591 because auto generated Microsoft.Management.Deployment -->
-    <!-- For our code, we still have stylecops acting on it. -->
+    <!-- For our code, we still have style cops acting on it. -->
```
Comment (florelis) [inline src/AppInstallerCLIE2ETests/AppInstallerCLIE2ETests.csproj:15]:
```suggestion
    <!-- For our code, we still have StyleCop acting on it. -->
```

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ![winget install wingetcreate](.github/images/WingetInstall.gif)
 
-If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
+If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "Winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
```
Comment (florelis) [inline README.md:7]: I believe the proper capitalization is WinGet or winget. In this case it refers to the source, so it should be `winget` as shown in `winget source list`. In some of the other cases you are changing, I believe it refers to the executable `winget.exe` so it still should be in lowercase.

**Comment by skanda890** [comment]
Tell suggestions in #4619.

**Pull request closed without merging**